frontend/src/components/Common/Button/Button.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import styles from './Button.module.css'; // Import the CSS file
```

Comment:
```suggestion
import styles from './Button.module.css';
```

(We don't need comments like this since it's pretty obvious what this line does lol)